### PR TITLE
Search box

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Follow up of #54244.

r? @QuietMisdreavus 

screen shot 2018-09-16 at 21 23 05

### CODE REVIEW

File: src/doc/index.md [hunk 1]
```diff
@@ -45,8 +62,9 @@ accomplishing various tasks.
 
 <div>
   <form action="std/index.html" method="get">
-    <input type="search" name="search"/>
-    <button>Search</button>
+    <input id="search-input" type="search" name="search"
+           placeholder="Search into the standard library!"/>
```
Comment (QuietMisdreavus) [inline src/doc/index.md]: I'd prefer something like "Search through the standard library" (without the exclamation mark), though i'm welcome to a second opinion from @rust-lang/docs.

File: src/librustdoc/lib.rs [hunk 2]
```diff
@@ -404,8 +404,11 @@ fn main_args(args: &[String]) -> isize {
                                   `short` (instead was `{}`)", arg));
         }
     };
+    let treat_err_as_bug = matches.opt_strs("Z").iter().any(|x| {
+        *x == "treat-err-as-bug"
+    });
```
Comment (QuietMisdreavus) [inline src/librustdoc/lib.rs]: This branch includes your `-Z treat-err-as-bug` branch as well. Please remove that commit from this branch.

**Review by QuietMisdreavus — CHANGES_REQUESTED** [review]

File: src/librustdoc/lib.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply src/librustdoc/lib.rs]: Woups, indeed!

File: src/doc/index.md (hunk 1 above)
Comment (GuillaumeGomez) [reply src/doc/index.md]: Yes sir!

**Comment by GuillaumeGomez** [comment]
Updated.

**Review by QuietMisdreavus — APPROVED** [review]

**Comment by QuietMisdreavus** [comment]
Unless someone from docs comes up with a better search-box placeholder text, r=me if travis is green.

**Pull request merged**